projects/laji/src/app/shared-modules/observation-map/observation-map/observation-map.component.ts

```diff
@@ -539,20 +521,25 @@ export class ObservationMapComponent implements OnInit, OnChanges, OnDestroy {
       on: {
         click: this.onFeatureClick.bind(this)
       },
-      marker: {
-        icon: getPointIcon
+      cluster: {
+        singleMarkerMode: true,
+        maxClusterRadius: 15,
+        iconCreateFunction: this.getClusterIcon.bind(this),
+        showCoverageOnHover: false
       },
       label: this.translate.instant('observation.map.dataOpacityControl.label'),
       visible: this.dataVisible,
       opacity: this.opacity,
       maxFillOpacity: 0.8,
+      controlFillOpacity: true,
       onOpacityChange: this.onOpacityChange.bind(this),
       onVisibleChange: this.onDataVisibleChange.bind(this)
     };
   }
 
   private onOpacityChange(opacity: number) {
     this.opacity = opacity;
+    this.lajiMap.map.getData()[1].groupContainer.refreshClusters();
```

Comment:
Laji-map handles updating the cluster opacity internally already, so this is redundant